git-link.el

```diff
@@ -35,6 +35,10 @@
 
 ;;; Change Log:
 
+;; 2022-12-27 - v0.8.7
+;; * Add support for [gitiles](https://gerrit.googlesource.com/gitiles) (Issue #100)
```

Comment:
Any reason for using Markdown here?

Reply:
No reason in particular, I thought the link would be helpful. I removed these comments anyway, per the thread above :+1:

Reply:
I just don't know of anything that will render the Markdown so just extra syntax.